util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -102,6 +102,13 @@ ${make_ral_pkg_window_class(dv_base_names.mem, esc_if_name, window)}
       inst_decl = f'{inst_name}[{count}]' if count > 0 else inst_name
 %>\
     rand ${reg_type} ${inst_decl};
+      % if r0.alias_target is not None:
+<%
+        alias_inst_name = r0.alias_target.lower()
+        alias_inst_decl = f'{alias_inst_name}[{count}]' if count > 0 else alias_inst_name
+%>\
+    rand ${reg_type} ${alias_inst_decl}; // aliases to ${inst_decl}
```

Comment:
Possibly silly question, but what is there to randomise on the alias register?

Reply:
Hm... I just used the same type for this variable as for `rand ${reg_type} ${inst_decl};` since both are supposed to point to the same object. I am not sure whether this would be required, though. Maybe @weicaiyang knows?